### DISCUSSION TITLE
Quebra de linha no text

### DISCUSSION THREAD

**Discussion by dayangomes** [kyechan99/capsule-render#15] [Q&A]
Tem como eu eu dar uma quebra de linha no título? 
Nesse exemplo, quero deixar o "Olá" em cima e quebrar o resto do texto para baixo (Ps: Não quero usar o &desc)
&text=Olá,%20me%20chamo%20Dayan%20Ramos%20Gomes

**Comment by kauer3** [comment]
Maybe we should try to communicate in English, since it's what @kyechan99 uses here.

**Comment by kauer3** [comment]
I understand why you don't want to use `desc`, because although you can make them the same size with `fontSize` and `descSize`, `desc` can't be as bold as the main text.
And I don't think there's a way to add a line break to the text today, since we pass it as a url parameter. However, it shouldn't be hard to implement, either adding another parameter to put a line of text between `text` and `desc` or getting some character(s) (e.g. \n) to be replaced by `<br>` in the api. The latter would have the benefit of accepting multiple line breaks. I could work on a pr if @kyechan99 agrees with it.

**Comment by kyechan99** [comment]
Cool :)

First, **I think** there is no way for a line break in the current version.
So we need to write new code for this.
If you have a good way, please send me a PR. 🤩 
The way to replace the `<br>` tag also seems very interesting.

**Comment by kauer3** [comment]
I sent the PR #17. It was harder than I thought, because I cannot place a `<br>` tag inside the svg, so I had to generate a `text` tag dynamically for each line and adjust the position with the `x` param. However, I think the final result turned out even better! Once you have a chance, take a look and let me know what you think.
